### DISCUSSION TITLE
Calls to Action for Better Patch Adoption

### DISCUSSION THREAD

**Discussion by epicfaace** [PlaintextGroup/oss-virtual-incubator#35] [Proposal Discussion]
Discuss https://github.com/PlaintextGroup/oss-virtual-incubator/blob/main/proposals/calls-to-action-for-better-patch-adoption.md

**Comment by lucasgonze** [comment]
Let's discuss this.

Topic: scope. How is this innovative and ambitious enough to systemically improve the sustainability of the OSS ecosystem, and yet small enough to be accomplishable within the budge of a mini-grant? it is intended to balance these constraints.

The problem being addressed is plainly a systemic improvement to OSS as a whole. Failures to adopt patches are a major source of security problems across the OSS ecosystems, and they compound as insecurity in one component can be combined with insecurities in other components. 

Yet is it is small and targeted. The output is a high-quality understanding of the causes of the problem for the purposes of making future improvements. The budget of a mini-grant is a perfect fit. Given this output, future phases of work can be tackled incrementally, with a similarly big scope but small budget.
